src/panel.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Block-facebook</title>
+    <link rel="stylesheet" href="panel.css">
+  </head>
+  <body>
+    <div id="panel-1" class="wrapper">
+      <h4 class="profile heading padding">
+        Facebook Container
+      </h4>
+      <p class="padding">
+        Facebook Container has been installed and is preventing Facebook from tracking you around the web. Anytime you see <img src="fb-ban2.png"> this addon is blocking Facebook from traking you.
```

Comment:
> from traking you

typo: "tracking"

Reply:
I am so sorry for typos I am correcting them. I just typed them in hurry.